doc/Environments.md

```diff
@@ -14,17 +14,14 @@ By default, the template offers the following runtime environments, defined by t
 
 You can add / remove runtime environments by simply adding or removing appsettings files (e.g. `appsettings.myenvironment.json` will create a new runtime environment named `myenvironment`).
 
-This is configured inside the [AppSettingsConfiguration.cs](../src/app/ApplicationTemplate.Shared/Configuration/AppSettingsConfiguration.cs) file.
+This is governed by `IEnvironmentManager` and configured inside the [ConfigurationConfiguration.cs](../src/app/ApplicationTemplate.Presentation/Configuration/ConfigurationConfiguration.cs) file.
```

Comment:
Isn't this already in `main`? Why are we seeing these changes again if they're already there?